karma.conf.js

```diff
@@ -1,77 +1,43 @@
-// Sample Karma configuration file, that contain pretty much all the available options
-// It's used for running client tests on Travis (http://travis-ci.org/#!/karma-runner/karma)
-// Most of the options can be overriden by cli arguments (see karma --help)
-//
-// For all available config options and default values, see:
-// https://github.com/karma-runner/karma/blob/stable/lib/config.js#L54
-module.exports = function(config) {
-  config.set({
+process.env.BUILD_TEST = 'true'
+var webpackConfig = require('./webpack.config')
+delete webpackConfig.entry
+delete webpackConfig.output
```

Comment:
> Because it's modifying the original object?

yes. Karma is running in the same node process as webpack and therefore everything you do without a restart uses that config config object.